### DISCUSSION TITLE
Quicktime Player recording?

### DISCUSSION THREAD

**Discussion by MaikWaschfeld** [ExistentialAudio/BlackHole#390] [Q&A] [answered]
I did some testing with BlackHole, 16ch. from QLab and the Quicktime Player's "New Audiorecording" as destination.
With quality setting on "Maximum", all went as I expected it: one 16-ch. aifc-file.
While using quality setting on "High", Quicktime Player ended with an m4a-file with 8 tracks and not honoring the source-assignment at all, so all the separate tracks from BlackHole ended on channel 1 of the m4a-file.
Is this expected behaviour?

With kindest regards...
...Maik

**Comment by devinroth** [comment]
Expected behaviour of QuickTime. Just use the 2ch version so QuickTime doesn’t get confused.

**Comment by MaikWaschfeld** [comment]
Using the 2ch version doesn't get me the 16 separate channels, I was looking for.
Or will it?

**Comment by devinroth** [comment, marked as answer]
It won’t. But Quicktime also won’t get you 16 channels. That’s a limitation of Quicktime.

You need to record with something else. Use Garage Band or Audacity.

Devin Roth
